test/unit/mongo/MongoTripodTest.php

```diff
@@ -740,12 +738,21 @@ public function testSaveChangesWithInvalidCardinality()
         $config = array();
         $config['namespaces'] = array('rdf'=>'http://www.w3.org/1999/02/22-rdf-syntax-ns#');
         $config["defaultContext"] = "http://talisaspire.com/";
-        $config["transaction_log"] = array("database"=>"transactions","collection"=>"transaction_log","connStr"=>"mongodb://talisaspire:acorn123@46.137.106.66:27018");
-        $config["es_config"] = array("search_document_specifications"=>array(),"indexes"=>array(),"endpoint"=>"http://localhost");
-        $config["databases"] = array(
-            "testing"=>array(
-                "connStr"=>"mongodb://localhost",
-                "collections"=>array(
+        $config["transaction_log"] = array("database"=>"transactions","collection"=>"transaction_log","data_source"=>"tlog");
+        $config["data_sources"] = array(
+            "db"=>array(
+                "type"=>"mongo",
+                "connection"=>"mongodb://localhost"
+            ),
+            "tlog"=>array(
+                "type"=>"mongo",
+                "connection"=>"mongodb://talisaspire:acorn123@46.137.106.66:27018"
```

Comment:
these are real credentials we probably need to remove these values and use some dummy ones~?